graphql-server/src/queryFactory/dolt/index.ts

```diff
@@ -458,21 +465,37 @@ export class DoltQueryFactory
     );
   }
 
-  async callPullRemote(args: t.PushOrPullRemoteArgs): t.PR {
+  async callPullRemote(args: t.RemoteMaybeBranchArgs): t.PR {
     return this.query(
       qh.callPullRemote,
       [args.remoteName, args.branchName],
       args.databaseName,
     );
   }
 
-  async callPushRemote(args: t.PushOrPullRemoteArgs): t.PR {
+  async callPushRemote(args: t.RemoteMaybeBranchArgs): t.PR {
     return this.query(
       qh.callPushRemote,
       [args.remoteName, args.branchName],
       args.databaseName,
     );
   }
+
+  async callFetchRemote(args: t.RemoteMaybeBranchArgs): t.PR {
+    return this.query(
+      qh.callFetchRemote(args.branchName),
+      [args.remoteName],
+      args.databaseName,
+    );
+  }
+
+  async callMergeBase(args: t.RefsArgs): t.PR {
```

Comment:
Should only use `call` for procedures
```suggestion
  async getMergeBase(args: t.RefsArgs): t.PR {
```